.github/workflows/main.yml

```
@@ -450,12 +453,23 @@ jobs:
 
       - name: Build all code
         run: make build
+      - name: Set up Prettier and ESLint cache
+        uses: actions/cache@v4
+        with:
+          path: |
+            node_modules/.cache/prettier/.prettier-cache
+            .eslintcache
+          key: ${{ runner.os }}-prettier-eslint-${{ hashFiles('yarn.lock', '.eslintignore', '.eslintrc.cjs', '.prettierrc.json', '.prettierignore') }}-${{ hashFiles('packages/*/src/**') }}
+
       - name: Run the JavaScript linter
-        run: make lint-js
+        run: |
+          git restore-mtime
+          make lint-js-cached
```

I'm assuming `git restore-mtime` is left over from a previous implementation?

```suggestion
        run: make lint-js-cached
```

---

It actually isn't -- there is a bug in the prettier cache. https://github.com/prettier/prettier/issues/17278

---

That definitely warrants a comment in the file!

---

Added in 2147209943d2f137c6ae85fce6e37dc5aa7072ff.